src/components/App.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPannel from './ButtonPanel';
+
```

Comment:
- [ ] As stated in the project's requirements, kindly import `calculate.js` in `App` component. [Screenshot from Pathwright](![image](https://user-images.githubusercontent.com/49621279/126829120-e56b925f-416c-42ce-b769-a5e9a7d1ff30.png)).